src/app/examples/pos/pos.html

```diff
@@ -23,8 +23,7 @@
 <!-- Success -->
 <div
   *ngIf="
-    (service.meow$ | async | isSuccess) ||
-    (service.meow$ | async | isInProgress)
+    (service.meow$ | async | isSuccess)
```

Comment:
That works correct, but probably we should improve the style and simplify the code to be just `<div *ngIf="service.meow$ | async | isSuccess">` (on a single line and without parentheses)